File: 27-RemoveElement/27-RemoveElement.cpp
// Last updated: 22/2/2026, 10:20:28 pm
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int  n=nums.size();
        int k=0;
        for(int i=0;i<n;i++){
            if(nums[i]!=val){
                nums[k++]=nums[i];
            }
        }
        return k;
    }
};

File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 22/2/2026, 10:20:32 pm
class Solution:
    def removeElement(self, nums: list[int], val: int) -> int:
        tmp = []
        for num in nums:
            if num == val:
                continue
            tmp.append(num)
        for i in range(len(tmp)):
            nums[i] = tmp[i]
        return len(tmp)